lab-nassir/app/view/home/home-controller.js

```diff
@@ -0,0 +1,26 @@
+'use strict';
+
+require('./_home.scss');
+
+module.exports = ['$log', '$rootScope', 'galleryService', HomeController ];
+
+function HomeController($log, $rootScope, galleryService){
+  $log.debug('Initializing HomeController;');
+
+  this.galleries = [];
+
+  this.fetchGalleries = function() {
+    $log.debug('Fetching gallleries;');
+    galleryService.fetchGalleries()
+    .then(galleries => {
+      this.galleries = galleries;
+    });
+  };
+
+  this.fetchGalleries();
+
+  $rootScope.$on('$locationChangeSuccess', () => {
+    this.fetchGalleries();
```

Comment:
Great work updating the galleries array/the view on home controller init and location change.